### PR TITLE
Fix issue with lack of query_counter when Query serialization.

### PR DESCRIPTION
**Pull request by cimadai**
## What is this pull request?
Fix issue with lack of query_counter when Query serialization.

## Details/Features
- Add set_query_counter method to query serialization process.

### CODE REVIEW

**Comment by cimadai** [comment]
OK, I'll add test.

**Comment by cimadai** [comment]
Added one test. please check it~

File: test/module/irohad/model/converters/pb_query_factory_test.cpp [hunk 1]
```diff
@@ -38,13 +38,27 @@ void runQueryTest(std::shared_ptr<Query> query){
...
+  auto accountId = "test";
   PbQueryFactory queryFactory;
   QueryGenerator queryGenerator;
-  auto query = queryGenerator.generateGetAccount(0, "123", 0, "test");
+  auto query = queryGenerator.generateGetAccount(createdTime, creatorAccountId, queryCounter, accountId);
   auto pb_query = queryFactory.serialize(query);
   ASSERT_TRUE(pb_query.has_value());
+  auto &pl = pb_query.value().payload();
+  auto &pb_cast = pb_query.value().payload().get_account();
+  ASSERT_TRUE(pl.created_time() == createdTime);
```
Comment (l4l) [inline test/module/irohad/model/converters/pb_query_factory_test.cpp]: Use `ASSERT_EQ` instead, like: `ASSERT_EQ(pl.created_time(), createdTime)`
same below

File: test/module/irohad/model/converters/pb_query_factory_test.cpp [hunk 2]
```diff
@@ -38,13 +38,27 @@ void runQueryTest(std::shared_ptr<Query> query){
...
+  ASSERT_TRUE(pl.creator_account_id() == creatorAccountId);
+  ASSERT_TRUE(pl.query_counter() == queryCounter);
+  ASSERT_TRUE(pb_cast.account_id() == accountId);
   auto res_query = queryFactory.deserialize(pb_query.value());
   ASSERT_TRUE(res_query.has_value());
+  ASSERT_TRUE((*res_query)->created_ts == createdTime);
```
Comment (l4l) [inline test/module/irohad/model/converters/pb_query_factory_test.cpp]: Introduce an additional variable so you don't need to dereference every time

File: test/module/irohad/model/converters/pb_query_factory_test.cpp [hunk 3]
```diff
@@ -38,13 +38,27 @@ void runQueryTest(std::shared_ptr<Query> query){
...
+  ASSERT_TRUE((*res_query)->creator_account_id == creatorAccountId);
+  ASSERT_TRUE((*res_query)->query_counter == queryCounter);
+  ASSERT_TRUE(((iroha::model::GetAccount&)(**res_query)).account_id == accountId);
```
Comment (l4l) [inline test/module/irohad/model/converters/pb_query_factory_test.cpp]: It's better not use c-style cast, use:
```cpp
ASSERT_EQ(static_cast<GetAccount*>(*res_query)->account_id, accountId)
```

File: test/module/irohad/model/converters/pb_query_factory_test.cpp [hunk 4]
```diff
@@ -38,13 +38,27 @@ void runQueryTest(std::shared_ptr<Query> query){
 }
 
 TEST(PbQueryFactoryTest, SerializeGetAccount){
+  auto createdTime = 111u;
```
Comment (l4l) [inline test/module/irohad/model/converters/pb_query_factory_test.cpp]: Pleas follow the [codestyle](http://google.github.io/styleguide/cppguide.html#Variable_Names), rename to `created_time`
Same for the rest consts

**Pull request merged**